networkx/algorithms/bipartite/tests/test_generators.py

```diff
@@ -329,6 +347,12 @@ def test_preferential_attachment(self):
         assert not G.is_multigraph()
         assert not G.is_directed()
 
+        pytest.raises(
```

Comment:
I realized this is just copying the pattern of those below, but I'd vote to use the context manager pattern regardless.

Reply:
Just for this test or would it be better to change all of them?